### ISSUE TITLE
Skyline vector

### ISSUE THREAD

**Issue by emilyhobson** [cityenergyproject/dc#2]

**Comment by alecburch** [comment]
I found a good vector skyline, just need to decide what color. Unless we want to use SVG, which I'd be ok with.

**Comment by alecburch** [comment]
@ranchodeluxe here are the skyline images for the navigation. 

![dc-skyline](https://cloud.githubusercontent.com/assets/5263476/17183853/33c627c4-53de-11e6-84da-17fb9043c170.png)
![dc-skyline 2x](https://cloud.githubusercontent.com/assets/5263476/17183854/33d6f8d8-53de-11e6-828f-974647ab6b4a.png)

**Comment by ranchodeluxe** [comment]
@alecburch : is there anyway I could get the skyline combined with the DOEE logos in [Google Drive](https://drive.google.com/drive/u/1/folders/0B5tBIGe_xerccU1vQzMwak1hMXc) in a single PNG like the Chicago site uses in the header? 

The Chicago site provides the same images with two different dimensions ( i assume for different screen resolutions ). You can peek at them here:

http://cityenergyproject.github.io/chicago/images/chicago@2x.png

http://cityenergyproject.github.io/chicago/images/chicago.png

**Comment by alecburch** [comment]
Yep, I can do that for you. Had to request access to the google drive folder. 

And yes, the 2 are for retina and non-retina screens.

**Comment by alecburch** [comment]
@ranchodeluxe here ya go

![dc-skyline](https://cloud.githubusercontent.com/assets/5263476/17302450/2f705cca-57d1-11e6-930a-c29022177c35.png)
![dc-skyline 2x](https://cloud.githubusercontent.com/assets/5263476/17302449/2f6fb72a-57d1-11e6-91de-ce54ea9ea419.png)

**Comment by emilyhobson** [comment]
They will send one over

**Comment by alecburch** [comment]
One what over, @emilyhobson?

**Comment by alecburch** [comment]
@emilyhobson si, si?

![dc_dc-skyline](https://cloud.githubusercontent.com/assets/5263476/17348622/52234538-58cc-11e6-9061-6e143bda0c28.png)
![dc_dc-skyline 2x](https://cloud.githubusercontent.com/assets/5263476/17348623/5224cc1e-58cc-11e6-913e-0fd33dd561b5.png)

**Comment by emilyhobson** [comment]
@alecburch Can you extract the middle one and review?

https://drive.google.com/open?id=0B5tBIGe_xercVGl1eTlsQ3BhaWo3czFIVU5KcklOSmtNSm1j

**Comment by emilyhobson** [comment]
New logos and skyline (?) https://drive.google.com/open?id=0B5tBIGe_xercaDdsMlpkMmNhVjQ

**Comment by alecburch** [comment]
@ranchodeluxe, here are all the files for the header. "dc-logo.png" will the left aligned in the header and "dc-skyline.png" will be right aligned. I have the padding in the images so you should be able to just drop them in. 

[dc header files.zip](https://github.com/cityenergyproject/dc/files/443460/dc.header.files.zip)
